extensions/quartz/deployment/src/main/java/io/quarkus/quartz/deployment/QuartzProcessor.java

```diff
@@ -75,17 +74,10 @@ AdditionalBeanBuildItem beans() {
     }
 
     @BuildStep
-    AdditionalBeanBuildItem jobs(CombinedIndexBuildItem combinedIndexBuildItem) {
-        // Jobs need to be marked as unremovable otherwise ArC will automatically remove them because they are not @Injected anywhere
-        AdditionalBeanBuildItem.Builder builder = AdditionalBeanBuildItem.builder().setUnremovable();
-        // Register Jobs
-        for (ClassInfo info : combinedIndexBuildItem.getIndex().getAllKnownImplementors(JOB)) {
-            if (Modifier.isAbstract(info.flags())) {
-                continue;
-            }
-            builder.addBeanClass(info.name().toString());
-        }
-        return builder.build();
+    AutoAddScopeBuildItem addScope() {
```

Comment:
Did not know of this build item. If I understand it correctly, it will automatically pick all the implementation of the given `interface`? Looks handy.

Reply:
It's a new stuff ;-). It aims to remove boilerplate necessary when annotation transformers are used to add a scope annotation to a class...

Reply:
It looks shiny ;-) 

Thanks. I'll keep it in mind.